featurebyte/query_graph/node/schema.py

```diff
@@ -250,3 +250,4 @@ class ColumnSpec(FeatureByteBaseModel):
 
     name: StrictStr
     dtype: DBVarType
+    description: Optional[str]
```

Comment:
This schema is used in query graph (https://github.com/featurebyte/featurebyte/blob/d21ce4a278e0f9607b5a7e6e1d343ce24db33682/featurebyte/query_graph/node/input.py#L41).

If we add the parameter here, it means changing of the column description will cause change in query graph. `ColumnInfo` already contains `description` attribute, curious why introduces the parameter here (https://github.com/featurebyte/featurebyte/blob/0660cbdc426a811681753fb11ab8f88d9e95e968/featurebyte/query_graph/model/column_info.py#L51).